ComponentLibrary/component-library.css

```diff
@@ -0,0 +1,289 @@
+    border: 3px solid var(--white);
+    font-size: 8px;
+    color: var(--green);
+    background-color: var(--green);
+    border-radius: 100%;
+    padding: 2px 4px;
+}
+.avatar-badge-lg {
+    position: relative;
+    right: 2rem;
+    bottom: 0.7rem;
+}
+.avatar-badge-md {
+    position: relative;
+    right: 1.7rem;
+    bottom: 0.5rem;
+}
+.avatar-badge-sm {
+    position: relative;
+    right: 1.2rem;
+    bottom: 0.3rem;
+}
+
+    /* -- dismiss -- */
+.dismiss {
+    cursor: pointer;
+    color: var(--white);
+    background-color: var(--primary-dark);
+    padding: 5px 7px;
+    border-radius: 100%;
+}
+
+/* Buttons */
+.btn {
+    text-decoration: none;
+    padding: 0.5rem 1rem;
+    border-radius: 1rem;
+    border: 3px solid var(--primary-color);
+    cursor: pointer;
+    margin: 0.5rem 0;
+}
+.btn:focus{
+    outline: none;
+}
+.primary-btn {
+    color: var(--white);
+    background-color: var(--primary-color);
+}
+.secondary-btn {
+    color: var(--primary-color);
+    background-color: var(--white);
+}
+.link {
+    text-decoration: none;
+    color: var(--primary-color);
+}
+.icon-btn {
+    text-decoration: none;
+    color: var(--primary-color);
+}
+.floating-btn {
+    position: fixed;
+    right: 1rem;
+}
+/* Card */
+.card {
+    width: 18rem;
+    text-align: center;
+}
+.text-left {
+    text-align: left;
+}
+.card-img {
+    max-width: 100%;
+}
+.product-details {
+    text-align: left;
+}
+.product-name {
+    font-weight: bold;
+    font-size: 20px;
+}
+.product-section-name {
+    color: var(--primary-grey);
+}
+.product-price {
+    font-weight: bold;
+    font-size: 24px;
+}
+.cart-or-wishlist {
+    margin-bottom: 1rem;
+}
+/* -- card badge -- */
+.card-with-badge {
+    position: relative;
+}
+.card-badge {
+    position: absolute;
+    top: 2rem;
+    left: 2rem
+}
+/* -- dismiss card -- */
+.dismiss-card {
+    position: absolute;
+    top: 1rem;
+    right: 1rem;
+}
+/* -- shadow card -- */
+.shadow-card {
+    box-shadow: 10px 10px 10px var(--primary-grey);
+}
+
+/* Image */
+.img {
+    max-width: 100%;
+}
+.round-img {
+    border-radius: 100%;
+}
+
+/* Input */
+.input-div {
+    padding: 0.5rem;
+}
+.input {
+    width: 10rem;
+    padding: 0.5rem;
+    border: 1px solid var(--primary-dark);
+    border-radius: 3px;
+}
+
+/* List */
+.list-inline {
+    display: flex;
+}
+.list-style-none {
+    list-style-type: none;
+}
+   /* stacked list */
+.stacked-list {
+    width: fit-content;
+}
+.stacked-list-item {
+    width: fit-content;
+}
+
+/* Notification */
+.notification {
+    border-bottom: 1px solid var(--primary-dark);
+    padding: 0 0.5rem;
+    width: 20rem;
+}
+.notification-via {
+    font-weight: bold;
+}
+.notification-for {
+    font-weight: bold;
+}
+
+/* Nav */
+.navbar {
+    background-color: var(--primary-color);
+    padding: 1px;
+    border-radius: 1rem;
+}
+.nav-pill {
+    padding: 0.5rem 1rem;
+    cursor: pointer;
+}
+.selected-nav-pill {
+    font-weight: bold;
```

Comment:
Something like ```nav-pill-selected``` would maintain consistency here throughout selectors.

Reply:
Ya I'll be changing selectors, soon. Also will need to use camelCasing for react.